lib/timers.js

```diff
@@ -502,24 +502,26 @@ Timeout.prototype.close = function() {
 };
 
 
-var immediateQueue = {};
-L.init(immediateQueue);
+var immediateQueue = L.create();
```

Comment:
I don't really understand why this change is needed or is helpful but I guess it does clean things up a little.